howto-build-android.md

```diff
@@ -116,6 +116,10 @@ Build the Monero libraries and their dependencies:
 
 `$ ./build_all.sh`
 
+Build the Decred library:
+
+`$ ./build_decred.sh`
```

Comment:
`build_decred.sh` should be in `build_all.sh`. Remember that a builder may want to build only monero and not include decred. The `build_all.sh` script does the needed checks and invokes the appropriate build scripts.

Reply:
I did similar for ios and macos in https://github.com/JoeGruffins/cake_wallet/pull/13/commits/55d837ccc2a1a42083b95d7d9794e4e184a170ea